src/core/internal/parseoptions.d

```diff
@@ -156,16 +173,68 @@ inout(char)[] find(alias pred)(inout(char)[] str)
     return null;
 }
 
-bool parse(T:size_t)(const(char)[] optname, ref inout(char)[] str, ref T res, const(char)[] errName)
+bool parse(T : size_t)(const(char)[] optname, ref inout(char)[] str, ref T res, const(char)[] errName)
+{
+    return parseSuff(optname, str, res, errName, false);
+}
+
+bool parseSuff(T : size_t)(const(char)[] optname, ref inout(char)[] str, ref T res, const(char)[] errName, bool mayHaveSuffix)
```

Comment:
You could simply add `mayHaveSuffix` as a default parameter to avoid this intermediate function (which isn't named appropriatly anyway because `mayHaveSuffix`  determines whether a suffix is accepted or not).

```suggestion
bool parse(T : size_t)(const(char)[] optname, ref inout(char)[] str, ref T res, const(char)[] errName, bool mayHaveSuffix = false)
```

Reply:
This leads to adding same parameter to all another parse() functions

Reply:
Not really, you only pass `mayHaveSuffix` if the option type is `size_t` which will match this overload anyway.

Reply:
Can you make PR for this PR?

Reply:
@MoonlightSentinel got your point, parseSuff removed. Thanks!

Reply:
Thanks, sorry if my initial suggestion was misleading.